### PR TITLE
Integrate the Mozilla Screenshots UI into Blipz.

### PR DESCRIPTION
**Pull request by wisniewskit**
See #127 for a summary of what this PR is doing.

Note that all of the new files going into the `/screenshots` subdirectory are unmodified copies of the ones generated by the webextension version of the Screenshots addon, save for a few CSS rules added to the bottom of `inlineSelectionCss.js` to hide bits of UI that we don't need.

In addition, `blank.html` had to be placed in the root directory where the Screenshots code is hard-wired to expect it. Similarly, a few icons and l10n messages had to be placed where they were placed.

The most interesting bits of the PR for our purposes are the `screenshotsLoader.js` script (which is a minimal custom bootstrapper to call up the Screenshots "selector" UI), and the changes to our `background.js` to manage the UI and handle the messages it sends back.

### CODE REVIEW

**Comment by miketaylr** [comment]
> In addition, `blank.html` had to be placed in the root directory where the Screenshots code is hard-wired to expect it. Similarly, a few icons and l10n messages had to be placed where they were placed.

lol ok

File: webextension/blank.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<html></html>
```
Comment (miketaylr) [inline webextension/blank.html]: QUIRKS MODE??!??!?!

Comment (miketaylr) [reply webextension/blank.html]: (kidding, this likely doens't matter)

File: webextension/screenshotsLoader.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    if (actualScreenshotsFrame) {
+      actualScreenshotsFrame.style.display = "";
+      actualScreenshotsFrame = undefined;
+    }
+    return true;
+  };
+
+  return exports;
+})();
+null;
```
Comment (miketaylr) [inline webextension/screenshotsLoader.js]: What are all these `null`s at the end of files?

Comment (wisniewskit) [reply webextension/screenshotsLoader.js]: Console warnings will be logged when a script is injected using `browser.tabs.executeScript` API, when the last object in the file isn't JSON-serializable -- it's treated like a return value by the API, basically. So leaving the last line as `null` is a common trick to silence those warnings.

File: webextension/blank.html (hunk 1 above)
Comment (wisniewskit) [reply webextension/blank.html]: Heh. It's what screenshots is using, so I left it as-is :)

File: webextension/screenshotsLoader.js (hunk 2 above)
Comment (miketaylr) [reply webextension/screenshotsLoader.js]: interesting, TIL thanks!

**Pull request merged**